flocker/control/test/test_persistence.py

```diff
@@ -479,7 +431,7 @@ def _build_node(applications):
     APPLICATIONS,
     # If we add this hint on the number of applications, Hypothesis is able to
     # run many more tests.
-    average_size=10,
+    average_size=3,
```

Comment:
I'll try that as alternative to the other two. They definitely combined cut time from 60 seconds to 36 seconds on my machine.